packages/terra-consumer-nav/src/components/card/Card.jsx

```diff
@@ -0,0 +1,30 @@
+// temp card component for logo
```

Comment:
just wanna confirm that this card component will be replaced by this jira: https://jira2.cerner.com/browse/PORTALDEV-33956